assets/js/vue/components/stats/RepositorySummary.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+    <div id="program-resources">
+        <div style="display:table-row;">
+            <div class="summaryLabel" style="display:table-cell;">Repository Summary</div>
+        </div>
+        <div style="display:table-row;">
+            <div style="display:table-cell;">
+                <table>
+                    <tr>
+                        <td>
+                            <img src="~images/icon-datasets.png">
+                            <div class="count">{{datasets}}</div>
+                            <div class="label">Datasets</div>
+                        </td>
+                        <td>
+                            <img src="~images/icon-filesize.png">
+                            <div class="count">{{totalsize}}</div>
+                            <div class="label">TB Of Data</div>
+                        </td>
+                        <td>
+                            <img src="~images/icon-downloads.png">
+                            <div class="count">{{totalDownloads}}</div>
+                            <div class="label">Total Downloads</div>
+                        </td>
+                        <td>
+                            <img src="~images/icon-projects.png">
+                            <div class="count">{{researchGroups}}</div>
+                            <div class="label">Research Groups</div>
+                        </td>
+                        <td>
+                            <img src="~images/icon-researchers.png">
+                            <div class="count">{{people}}</div>
+                            <div class="label">People</div>
+                        </td>
+                    </tr>
+                </table>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+    export default {
+        name: "RepositorySummary",
+        data() {
+            return {
+                datasets: '-',
```

Comment:
This primarily defines the type of the variable, should we just use 0 here and use v-show=false till the data loads